synapse/lib/link.py

```diff
@@ -124,6 +130,19 @@ class Link(s_base.Base):
 
         self.onfini(fini)
 
+    def getSpawnInfo(self):
+        info = {}
+
+        # selectively add info for pickle...
+        if self.info.get('unix'):
+            info['unix'] = True
+
+        return {
+            'info': info,
+            # a bit dirty, but there's no other way...
```

Comment:
some deep magic here, but the socket itself can be serialized by the forking pickle code so we can hand it through